harvest-refactor/qnl-iiif-manifest-harvest.py

```diff
@@ -0,0 +1,15 @@
+from lxml import etree
+import glob
+import urllib.request
+
+ns = {'oai': "http://www.openarchives.org/OAI/2.0/"}
+
+files = glob.glob('/Users/jtim/Dropbox/DLSS/DLME/dlme-metadata/qnl/british-library/data/*.xml')
```

Comment:
Is there a way to do this as a relative path or maybe pass in the path as a parameter from the command line so you don't have to hardcode something specific to your laptop here?